ops/triton/triton_ragged_hstu_attention.py

```diff
@@ -691,10 +757,39 @@ def triton_ragged_attention(
     has_attn_scale = attn_scale is not None
     has_max_attn_len = max_attn_len is not None
 
-    grid = lambda meta: (  # noqa E731
-        triton.cdiv(N, meta["BLOCK_M"]),
-        Z * H,
-    )
+    TMA_SIZE = 128
+    BLOCK_D_V, BLOCK_D_Q = DimV, DimQ
+    desc_k = torch.empty((TMA_SIZE), device="cuda", dtype=torch.int8)
+    desc_v = torch.empty((TMA_SIZE), device="cuda", dtype=torch.int8)
+
+    def grid_tma(META):
+        nonlocal desc_k
+        nonlocal desc_v
+        k_buf = torch.empty_like(desc_k, device="cpu", pin_memory=True)
+        v_buf = torch.empty_like(desc_v, device="cpu", pin_memory=True)
+        triton.runtime.driver.active.utils.fill_2d_tma_descriptor(
+            k.data_ptr(),
+            L,
+            H * DimQ,
+            META['BLOCK_N'],
+            BLOCK_D_Q,
+            k.element_size(),
+            k_buf.numpy(),
+            META['enable_tma']
+        )
```

Comment:
This looks good except that we need to guard this whole section with META['enable_tma']. If we didn't enable TMA, we don't need to create the descriptor in k_buf/v_buf and copy over to desc_k/desc_v. Thanks Puyan!